tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
+                 "dockerskeleton" \
+                 "nodejs6action"  \
+                 "python2action"  \
+                 "python3action"  \
+                 "swift3action"   \
+                 "java8action"
+do
+  docker tag "${docker_image_prefix}/${container}" "${dockerhub_image_prefix}/${container}:latest"
+  docker tag "${docker_image_prefix}/${container}" "${dockerhub_image_prefix}/${container}:${TRAVIS_BRANCH}-${TRAVIS_COMMIT::7}"
+  docker push "${dockerhub_image_prefix}/${container}"
+done
```

Comment:
@markusthoemmes What you mean? To do it as part of the gradle distDocker task, to build and push at the same time?
Using something like `-PdockerRegistry=docker.io  -PdockerImageTag=latest` ?

I was thinking:
1. on pushing only after success on running tests.
2. I want to have two tags pushed "latest" but also "master-$commit"
having a tag with a commit or build id allows someone to pinned to a certain snapshot, until we have proper semver version tag

I would look into how to do the push using gradle and see if I can figure it out.